c.c

```diff
@@ -1388,10 +1388,16 @@ static void skipToMatch (const char *const pair)
 	{
 		if (CollectingSignature)
 			vStringPut (Signature, c);
-		
 
 		if (c == begin)
 		{
+			char s = cppGetc();
+			if (s == '<' && s == c) {
+				// templates can still contain "<<" ot "<<="
+				break;
```

Comment:
Doesn't breaking out of here put the parsing in a weird state, as now it will go forward parsing while inside a template parameter list?

(Also completely missed that there was already a pull request for this when one of my users pinged me about this being a problem w/FastLED).